gated-infrastructure-deploy/gated-infrastructure-deploy.yml

```diff
@@ -21,10 +21,13 @@
     type: object
   - name: TerraformOutputVariables
     type: object
+  - name: terraformazurecontainer
+    value: ukhydrographicoffice/terraform-azure:1.3.9
 
 jobs:
   - deployment: planInfrastructure
     displayName: "Plan Infrastructure"
+    container: ${{variables.terraformazurecontainer}}
```

Comment:
```
    container: ${{variables.terraformazurecontainer}}
```

`variables` isn't correct. There are no variables in this file. Only parameters.